test/end-to-end/pages/dashboard/chooseApplication.js

```diff
@@ -1,31 +1,48 @@
 /* eslint-disable no-await-in-loop */
 'use strict';
 
-const content = require('app/resources/en/translation/dashboard');
+const dashboardEn = require('app/resources/en/translation/dashboard');
+const dashboardCy = require('app/resources/cy/translation/dashboard');
 const testConfig = require('config');
 
-module.exports = async function() {
+module.exports = async function(language ='en') {
     const I = this;
-
+    const dashboardContent = language === 'en' ? dashboardEn : dashboardCy;
     await I.checkPageUrl('app/steps/ui/dashboard');
+    const welshLinkText = await I.grabTextFrom('//a[@class =\'govuk-link language\']');
+    console.log('Dash Board Link Name::-->' + welshLinkText);
```

Comment:
I noticed there is a few console.log. do you need them? or should it be removed before merging?